contributor-docs/testing.md

```diff
@@ -104,10 +173,19 @@ the following command:
 script/test:e2e --grep @avt
 ```
 
+## Accessibility Tests
+
+To be written in the collaboration with Eric Bailey.
```

Comment:
Maybe instead of specifically mentioning Eric, we should just say "the accessibility team". It's possible that Eric will get pulled onto something else and somebody else will help us with the a11y tests.

Reply:
I think it is a good idea. I talked about this with Eric and asked if he can help us. This is why I mentioned his name here but in case he gets busy, someone else from the accessibility team can help us ❤️ 

Also, I created [a separate ticket for it](https://github.com/github/primer/issues/1509) because I think there is some good amount of writing work we need to do there.

Reply:
➕ to this.. we should avoid referencing specific Hubbers directly in code or enduring documentation. Just referring to the Accessibility team or maybe a specific reviewer team like `@accessibility-reviewers` should provide enough clarity.

Reply:
I removed this section entirely as it was just there as a placeholder or a to-do. I created [an issue](https://github.com/github/primer/issues/1509) for it, therefore not needed anymore